src/chevah-python-test/test_python_binary_dist.py

```diff
@@ -464,17 +496,42 @@ def main():
     else:
         print('"uuid" module is present.')
 
+    try:
+        from charset_normalizer import from_path
+        tmp_results = from_path('../README.rst')
+    except:
+        sys.stderr.write('"charset-normalizer" is missing or broken.\n')
+        exit_code = 164
+    else:
+        print('"charset-normalizer" module is present.')
+
+    try:
+        from markupsafe import escape
+        tmp_text = escape('<script>alert(document.cookie);</script>')
+    except:
+        sys.stderr.write('"markupsafe" is missing or broken.\n')
+        exit_code = 165
+    else:
+        print('"markupsafe" module is present.')
+
     if os.name == 'nt':
         # Windows specific modules.
         try:
             from ctypes import windll
             windll
         except:
             sys.stderr.write('"ctypes - windll" is missing.\n')
-            exit_code = 152
+            exit_code = 171
         else:
             print('ctypes %s' % (ctypes.__version__,))
 
+        try:
+            import win32service
+            win32service.EnumWindowStations()
+        except:
+            sys.stderr.write('"pywin32" missing or broken.\n')
```

Comment:
We don't really use win32service in the code.

win32security security should be a better test, and we should show the error in the logs.

```suggestion
            import win32security
            win32security.win32api.GetCurrentThread()()
        except Exception as error:
            sys.stderr.write('"pywin32" missing or broken.\n {}'.format(error))
```

I will try this patch.